.gitlab-ci.yml

```diff
@@ -50,6 +51,65 @@ ghc-head:
     - echo "GHC head currently disabled, because singletons doesn't build on ghc head"
     - exit 1
 
+hackage-sdist:
+  extends: .tests
+  variables:
+    GHC: ghc-8.8.1
+  script:
+    - export GHC="${GHC:-$CI_JOB_NAME}"
+    - export THREADS=$(nproc)
+    - export CABAL_JOBS=$(nproc)
+    - export
```

Comment:
Should we have this export print our secrets and rely on gitlab to filter them out?

Reply:
You could unset them in a subshell around export:
`    - ( unset SNAPCRAFT_LOGIN_FILE HACKAGE_PASSWORD ; export )`
Or just drop the export.

Reply:
We _should_ be able to, but yeah, dropping the export seems a lot safer.